src/services/dom/errors.ts

```diff
@@ -8,6 +6,7 @@
  *
  */
 
+// eslint-disable-next-line max-classes-per-file
```

Comment:
I think this only throws a warning, not an error - can be removed

Reply:
It makes the file very tough to read. The entire thing is underlined in yellow